### PR TITLE
software-stack/lab: Remove inline html

### PR DESCRIPTION
**Pull request by Alex-deVis**
The use of inline html for labs leads to invalid paths on the website.
`.svg` files were updated to closely match the intended effect.

Signed-off-by: Alex Apostolescu <alexx.apostolescu@gmail.com>

### CODE REVIEW

File: content/chapters/software-stack/lab/content/modern-sw-stack.md [hunk 1]
```diff
@@ -2,7 +2,7 @@
 
 Most modern computing systems use a software stack such as the one in the figure below:
 
-<img src="../media/modern-sw-stack.svg" alt="Modern Software Stack" width="40%" />
+![Modern Software Stack](../media/modern-sw-stack-2.svg)
```
Comment (teodutu) [inline content/chapters/software-stack/lab/content/modern-sw-stack.md:5]:
```suggestion
![Modern Software Stack](../media/modern-sw-stack.svg)
```

**Review by teodutu — CHANGES_REQUESTED** [review]
I think I'm OK with this. The only issue I see with adjusting image sizes to fit our deployment is that they become less "generic" and other teachers might have a (slightly) harder time using them for their own classes. However, I believe this is a small price to pay for getting rid of those ugly (and broken) HTML links.

@razvand you proposed HTML links. Did you have anything else in mind?

**Review by teodutu — APPROVED** [review]
Fine by me I guess. Let's see what @razvand has to say.

**Review by razvand — CHANGES_REQUESTED** [review]
@Alex-deVis, be sure to merge the two commits.

Not sure what is the issue with the `<img src ...` construct. It seems to be working for slides (see [here](https://github.com/open-education-hub/operating-systems/blob/master/content/chapters/app-interact/lecture/slides/classification.md#gitlab) - rendered [here](https://open-education-hub.github.io/operating-systems/Lecture/Application-Interaction/)). Is this something that has only to do with the lab content, not the slides?

**Comment by Alex-deVis** [comment]
Hi, @razvand, you are right - this issue only affects lab content.

It has something to do with the paths where images are stored. In case of slides, the directory structure is the same - you still have a `media/` folder. In case of labs, all media is placed in `/assets/images`, so media paths have to be updated in the `.md`. The path is not updated for images referenced with inline html (``) and thus the undefined references occur.

I think the best approach would be to give up on inline html and update size from the `.svg` itself.

**Review by RazvanN7 — APPROVED** [review]

**Comment by RazvanN7** [comment]
I'll just merge this so that the broken links are fixed. If there are other issues with this PR, we can address them in subsequent PRs.

**Pull request merged**